.env.template

```diff
@@ -0,0 +1,63 @@
+# vim: ft=bash
+
+#### Cluster
+# Separate kubeconfig for your RHTAP cluster; `oc login` will create it automatically
+# Set this to ~/.kube/config if you'd rather use the shared kubeconfig
+export KUBECONFIG=/tmp/kubeconfig.rhtap.yaml
+
+#### Templates
+# For testing your changes, replace with your fork and branch of tssc-sample-templates
+export DEVELOPER_HUB__CATALOG__URL=https://github.com/redhat-appstudio/tssc-sample-templates/blob/main/all.yaml
+
+#### Github
+# https://docs.redhat.com/en/documentation/red_hat_trusted_application_pipeline/1.2/html-single/installing_red_hat_trusted_application_pipeline/index#creating_a_github_personal_access_token
+export GITOPS__GIT_TOKEN=
+### Option 1: create a new github app via rhtap-cli (https://github.com/redhat-appstudio/rhtap-cli)
+#
+# cd ../rhtap-cli
+# make
+# ./bin/rhtap-cli --kube-config "$KUBECONFIG" integration github-app --create "$my_app_name" --org "$my_gh_org" --token "$GITOPS__GIT_TOKEN"
+#
+# ⭐ Leave github_enabled=false for option 1, set to true for option 2
+export github_enabled=false
+### Option 2: you already have a github app
+# The easiest way to get a github app is to use option 1 and save the relevant variables for later reuse
+# Use hack/get-github-app-env.sh to get the relevant variables
+export GITHUB__APP__ID=
+export GITHUB__APP__CLIENT__ID=
+export GITHUB__APP__WEBHOOK__SECRET=
+export GITHUB__APP__CLIENT__SECRET=
+export GITHUB__APP__PRIVATE_KEY=
+
+#### Quay
+# You can usually just use the public instance (https://quay.io)
+export quay_install_enabled=false
+# In quay.io, go to Account Settings -> Generate Encrypted Password -> Docker Configuration
+# Paste the auth file content here
+export QUAY__DOCKERCONFIGJSON=
+# 1. Create a quay.io organization if you don't have one
+# 2. Go to the organization's page -> Applications and create an application
+# 3. Go to the application's page -> Generate Token
+#    Select these permissions: View all visible repositories
+#                              TODO ^ probably wrong
+export QUAY__API_TOKEN=
```

Comment:
Does anyone know what permissions the token needs? I haven't even figured what the token is for to be honest :smile:

Reply:
I remembered RHTAP has customer docs. (Also, wow I actually got the permissions right)